### PR TITLE
fix-container-registry

### PR DESCRIPTION
**Pull request by calvix**
workaround for the registry issue, tested on golem with 300 pulls, seems to working

### CODE REVIEW

**Review by tuladhar — APPROVED** [review]

File: helm/cluster-aws/templates/_helpers.tpl [hunk 1]
```diff
@@ -105,6 +105,13 @@ room for such suffix.
 {{- end -}}
 {{- end -}}
 
+{{- define "registryWorkaroundCommands" -}}
```
Comment (QuentinBisson) [inline helm/cluster-aws/templates/_helpers.tpl:108]: Does it make sense to document why this workaround is actually needed so we do not forget?

Comment (calvix) [reply helm/cluster-aws/templates/_helpers.tpl:108]: good point

Comment (tuladhar) [reply helm/cluster-aws/templates/_helpers.tpl:108]: @QuentinBisson It's documented here: https://github.com/giantswarm/roadmap/issues/1737

Comment (calvix) [reply helm/cluster-aws/templates/_helpers.tpl:108]: i added comment line as well

Comment (QuentinBisson) [reply helm/cluster-aws/templates/_helpers.tpl:108]: Sure but I meant in this PR :) It can be a simple comment to link the roadmap issue ;)

Comment (QuentinBisson) [reply helm/cluster-aws/templates/_helpers.tpl:109]: Loving it  @calvix :)

**Pull request merged**